citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -25,6 +41,9 @@
 			{
 			window.location.href = "../../items/" + key +"?index=" + index + "&page=[(${currentPage})]&sortBy=[(${sort})]&collectionId=[(${collectionId})]";
 			}
+		else if (currentURL.match(/search/)) {
+			window.location.href = "items/" + key + "?searchTerm=[(${searchTerm})]" + "&index=" + index + "&page=[(${currentPage})]&sortBy=[(${sort})]&collectionId=[(${collectionId})]";
+		}
 		else{
```

Comment:
let's make all these javascript if/else statements thymeleaf conditions; otherwise if we change the url or something like that, this javascript will stop working. It will also create less complex rendered html pages; and then use the code to get all the query parameters for all links (here and above for collection, search and other).